native/cocos/base/LogRemote.cpp

```diff
@@ -74,13 +74,16 @@ enum class UdpLogClientState {
     OK,
     DONE, // FAILED
 };
+
+uint64_t logId = 0;
```

Comment:
define it in anonymous namespace?

Reply:
Yes, it's already in an anonymous namespace

Reply:
ok